src/main/daml/Daml/Finance/Util/Date/DayCount.daml

```diff
@@ -223,3 +253,17 @@ isEndOfFebruary (y, m, d) =
 getDateEndOfFebruary : Int -> Date
 getDateEndOfFebruary y =
   fromGregorian (y, Feb, monthDayCount y Feb)
+
+-- | HIDE
+leapDayInDateRange : Int -> Date -> Date -> Bool
+leapDayInDateRange y fromDate toDate =
+  endOfFebDate >= fromDate && endOfFebDate <= toDate && isLeapYear y
```

Comment:
This is more efficient as typically the year is not a leap year (which means that you return `false` on the first predicate and do not evaluate the second and third)

```suggestion
  isLeapYear y && endOfFebDate >= fromDate && endOfFebDate <= toDate 
```

Reply:
Also, usually these DCF calculations exclude either the `fromDate` or the `toDate` (resp. "exclusive-inclusive" or "inclusive-exclusive"). It might be that the correct formulation should be either

```suggestion
  isLeapYear y && endOfFebDate > fromDate && endOfFebDate <= toDate
```

or

```suggestion
  isLeapYear y && endOfFebDate >= fromDate && endOfFebDate < toDate
```

Reply:
Currently, I only have this definition: 
`Method whereby interest is calculated based on the actual number of accrued days in the interest period, excluding any leap day from the count, and a 365-day year.`

I will have a closer look at the count logic.

Reply:
I added a few tests and changed to "exclusive-inclusive", which I think makes most sense for these examples.

Reply:
Sounds good! I would add a comment in the DCF type definition that this is exclusive-inclusive